source/features/forked-to.tsx

```diff
@@ -42,6 +55,11 @@ function createLink(baseRepo: string): string {
 		branch: 'HEAD'
 	});
 
+	if (pageDetect.isEditingFile()) {
+		url.route = 'blob'; // Replace the 'edit'
```

Comment:
Why?

Reply:
Can't check if the file exists with a route of edit??

Reply:
`doesFileExist` doesn't even use `url.route`, what are you talking about? 
https://github.com/sindresorhus/refined-github/blob/6f4ac06f08b5f2aea453dc4be5c50824b7cda2af/source/github-helpers/does-file-exist.tsx

Reply:
It takes the full url. If the route is Edit it will fail. 

I am not sure what you are not understanding.

Reply:
I remember why I did it. If the file does not exist the link will be a 404. Since the route is `edit`

Reply:
If the file doesn't exist it will be a 404 regardless of the route